### ISSUE TITLE
GitHub Packages

### ISSUE THREAD

**Issue by generik0** [castleproject/Windsor#519]
HI @jonorossi 
Looks like Github has added a "packages" functionality. So you can push packages to github.
Could it be interesting for prereleases etc? Now that the team city = gone.
Something to look into for the build?

**Comment by jonorossi** [comment]
Yep, GitHub Packages has been there since last year but I've not played with it yet. Definitely something to look to replace the AppVeyor NuGet feed. I already started writing a GitHub Actions "workflow" to move from AppVeyor, that can be a separate step afterwards.
